### PR TITLE
add metadata and fix some typo in the css

### PR DESCRIPTION
**Pull request by SilvestriStefano**
add metadata in the following pages:
+ index.rst
+ examples.rst
+ theory.rst (<-- needs description improvement)
+ get_started.rst (<-- needs description improvement)

add open graph metadata via package `sphinxext.opengraph` (check conf.py).

add missing `;` in css.

### CODE REVIEW

**Comment by SilvestriStefano** [comment]
For more information about metadata description see : [https://moz.com/learn/seo/meta-description](url)

File: docs/index.rst [hunk 1]
```diff
@@ -3,10 +3,20 @@
    You can adapt this file completely to your liking, but it should at least
    contain the root `toctree` directive.
 
+.. meta::
+   :format-detection: telephone=no
+   :robots: index, follow
+   :description: Sarkas is a fast pure-Python molecular dynamics suite for non-ideal plasmas.
```
Comment (lucianogsilvestri) [inline docs/index.rst:9]: Please change description to : Sarkas: A Fast pure-Python Molecular Dynamics suite for Plasma physics

**Review by lucianogsilvestri — CHANGES_REQUESTED** [review]
Thanks a lot for your improvements. Could you make few changes before we merge it ?

File: docs/index.rst (hunk 1 above)
Comment (SilvestriStefano) [reply docs/index.rst]: done.

File: docs/theory/theory.rst [hunk 2]
```diff
@@ -1,3 +1,12 @@
+
+.. meta::
+   :format-detection: telephone=no
+   :robots: index, follow
+   :description: Theory behind Sarkas.
```
[the remark this answers is not in the document]
Comment (SilvestriStefano) [reply docs/theory/theory.rst]: done.

**Pull request merged**